setup-production.adoc

```diff
@@ -358,14 +358,14 @@ Map the file into `/taiga-back/settings/config.py`. Have in mind that you have t
 
 .Map a `conf.json` file
 
-From https://github.com/kaleidos-ventures/taiga-front[taiga-front] download the file `dist/conf.example.json` and rename it:
+From https://github.com/kaleidos-ventures/taiga-front[taiga-front] download the file `conf/conf.example.json` and rename it:
 
 [source,bash]
 ----
-mv dist/conf.example.json dist/conf.json
+mv conf/conf.example.json conf/conf.json
```

Comment:
this change is ok, given the original code, but it really should be:
`mv conf.example.json conf.json`